corehq/apps/reports/standard/cases/basic.py

```diff
@@ -80,6 +80,12 @@ def _build_query(self):
                 commtrack=False,
             )
             query = query.NOT(case_es.owner(ids_to_exclude))
+        elif self.request.can_access_all_locations and EMWF.show_deactivated_data(mobile_user_and_group_slugs):
+            owner_ids = (user_es.UserES()
```

Comment:
Is it not possible to use `get_user_ids_for_user_types` here or extend it if not?